README.md

```diff
@@ -73,9 +73,17 @@ where
 
 It is possible for multiple localServerName environment variables to be specified that each point to different 'view servers', this is the multi tenant support. The server name being the tenant name. 
 
-Example:
-EGERIA_PRESENTATIONSERVER_SERVER_aaa={"remoteServerName":"cocoView1","remoteURL":"https://localhost:9443"}
-
+Since this environment variable is typically handled by the shell and includes a json fragment, when setting it you need to be sure
+to 'escape' quotation characters, so you would type:
+
+```bash
+EGERIA_PRESENTATIONSERVER_SERVER_aaa="{\"remoteServerName\":\"cocoView1\",\"remoteURL\":\"https://localhost:9443\"}"
```

Comment:
AARRRGGHHH escaping escapes

Reply:
@planetf1 I do not think it is right to say that "this environment variable is typically handled by the shell". This is true in production or container environments, but is confusing if running as a developer, as the escaping is not required when the content is picked up from the .env file. The example as it was is exactly what is required for the .env file content. I suggest adding the new content rather than replacing the existing example and having words to explain that one is for the .env file content and the other is an escaped version is running from a script.